### PR TITLE
(itests, test-framework) Move Project creation logic from requirement to separate class.

### PR DESCRIPTION
**Pull request by rhopp**
So this logic can be consumed by tests.

# Pull Request Checklist
## General
* Is this a blocking issue or new feature? If yes, QE needs to +1 this PR

## Code
* Are method-/class-/variable-names meaningful?
* Are methods concise, not too long?
* Are catch blocks catching precise Exceptions only (no catch all)?

## Testing
* Are there unit-tests?
* Are there integration tests (or at least a jira to tackle these)?
* Is the non-happy path working, too?
* Are other parts that use the same component still working fine?

## Function
* Does it work?

### CODE REVIEW

**Comment by adietish** [comment]
testPR

File: test-framework/org.jboss.tools.openshift.reddeer/src/org/jboss/tools/openshift/reddeer/utils/ProjectNativeUtils.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+import org.jboss.reddeer.common.wait.WaitUntil;
+import org.jboss.tools.openshift.core.connection.Connection;
+import org.jboss.tools.openshift.reddeer.condition.OpenShiftProjectExists;
+import org.jboss.tools.openshift.reddeer.requirement.OpenShiftResourceUtils;
+
+import com.openshift.restclient.ResourceKind;
+import com.openshift.restclient.model.IProject;
+import com.openshift.restclient.model.project.IProjectRequest;
+
+public class ProjectNativeUtils {
```
Comment (adietish) [inline test-framework/org.jboss.tools.openshift.reddeer/src/org/jboss/tools/openshift/reddeer/utils/ProjectNativeUtils.java]: I see the added value of extracting this away 
I find this class name confusing. Out of the blue I would have thought that these are utils for Eclipse "native" projects, aka workspace projects. Dont you think so?
My pick would be to either move these 2 methods into the existing OpenShiftResourceUtils or have the class called OpenShiftProjectUtils (which would also match the other naming scheme of the general purpose resource utils class OpenShiftResourceUtils). Dont you think so?

**Comment by adietish** [comment]
weird, jenkins fails to run, misses the activator in egit tests.

File: test-framework/org.jboss.tools.openshift.reddeer/src/org/jboss/tools/openshift/reddeer/utils/ProjectNativeUtils.java (hunk 1 above)
Comment (rhopp) [inline test-framework/org.jboss.tools.openshift.reddeer/src/org/jboss/tools/openshift/reddeer/utils/ProjectNativeUtils.java]: I'll address that.

**Comment by adietish** [comment]
testPR

**Comment by rhopp** [comment]
I've created package o.j.t.openshift.reddeer.utils.v3, where I moved OpenShiftResourceUtils  and ProjectNativeUtils. And I also renamed both of them to OpenShift3Native<Resource/Project>Utils. I wanted to be able to distinguish whether class uses UI, or core api from the class name and I didn't think of anything better than "Native" for the core-api ones.

**Pull request merged**